backend-rust/src/domain/rooms/message.rs

```diff
@@ -0,0 +1,39 @@
+#[derive(Clone)]
+pub struct ChatMessage {
+    sender: String,
+    message: String,
+    room_name: String,
+    sent_at: jiff::Zoned,
+}
```

Comment:
Quando a gente vem de uma lang OOP, é comum colocarmos os campos como private e usar getters/setters. Com o rust isso geralmente n tem mt valor. Eu raramente uso campos que não são pub, principalmente em executáveis, faz mais sentido em libs.

Se a referência pra struct for de leitura e não de mut, quem tá usando a lib já não vai poder alterar os valores de qualquer jeito. E outros lugares saem ganhando tb.

Por exemplo, o código em `backend-rust/src/domain/mappers.rs` poderia usar `message.sender` ao invés de `.sender().to_string()`. A versão atual clona a string inteira (uma alocação seguida de memcpy), e depois droppa a string original dentro de ChatMessage. Se o campo fosse público vc poderia só mover ela de lugar.